### ISSUE TITLE
Title with '&' character results in a package name with an '&' character.

### ISSUE THREAD

**Issue by awalterschulze** [NYTimes/openapi2proto#56]
Really great tool.  Thank you so much for this.

A json swagger file with

```
"title": "API to View & Search for Items",
```

Will result in a proto with 

```
package apitoview&searchforitems;
```

openapi2proto should probably take special characters into account?

**Comment by jprobinson** [comment]
Resolved in #57

**Comment by awalterschulze** [comment]
Thank you so much :)
Great fix and great test.
Really quality work.
